amaechi-chuks/iReporter
# bug(fix): enable eslint extension on my vscode

**What does this PR do?**
This PR sets up an eslint extension on my code editor to track my code.

Description of tasks:

Ensure that eslint is enable
Ensure that all indentation is fixed
Remove excessive comments
Remove whitespace

**How can this PR be tested**

Using any code editor of your choice
Run the code and check the following:
- Indentation
- whitespace
- excessive comments

**Link to relevant story** 👍 
#162383839
